tests/test_inference_client.py

```diff
@@ -872,57 +873,57 @@ def test_open_as_binary_from_bytes(self) -> None:
             "https://api-inference.huggingface.co/models/username/repo_name/v1/chat/completions",
         ),
         # Fal.ai endpoints
-        (
-            "fal-ai",
-            "text-to-image",
-            "username/repo_name",
-            "https://fal.run/username/repo_name",
-        ),
-        (
-            "fal-ai",
-            "automatic-speech-recognition",
-            "username/repo_name",
-            "https://fal.run/username/repo_name",
-        ),
+        # (
```

Comment:
Mid-term, would be cool to test providers separately. Typically not all need to implement the same interface (e.g. `build_url`) as long as `prepare_payload` is implement. Provider interfaces are quite simple and independent from sync/async clients